generators/app/utils.js

```diff
@@ -84,3 +85,28 @@ function extractEntitiesComponents(previousKafkaConfiguration) {
         consumers: extractConsumerEntitiesName(previousKafkaConfiguration)
     };
 }
+
+function orderKafkaRootPropertiesAsFirst(properties = {}) {
```

Comment:
Maybe rename it simply?

```suggestion
function sortKafkaProperties(properties = {}) {
```